assets/sass/cds/_components.scss

```diff
@@ -454,9 +557,188 @@
       width: 70%;
     }
   }
+	h1 {
+		font-size: 4.6rem;
+		font-weight: 100;
+		margin-top: 0;
+		margin-top: 1.5rem;
+		color: $black;
+		margin-left: 0;
+		margin-right: 0;
+		padding-left: 0;
+	}
+
+	h2{
```

Comment:
These change the meet-the-team page, I thought this was for the job posting page?